src/features/landing/components/AboutUs/Info/Info.tsx

```diff
@@ -0,0 +1,57 @@
+'use client';
+
+import { Stack, Button, Typography } from '@mui/material';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+function Info() {
```

Comment:
why did you move this component to another file? AboutUs/index.tsx looks not so difficult for moving this part to another component

Reply:
I didn't understand a bit what was meant